src/platform/SDL2Window.cxx

```diff
@@ -187,128 +181,110 @@ void SDLWindow::swapBuffers()
 bool SDLWindow::create(void)
 {
     int targetWidth, targetHeight;
```

Comment:
From a complexity standpoint, uninitialized variables are bad. If this block were refactored so that these variables were initialized to the base dimensions, and overridden if `fullScreen` is true (if case below, lose the else), then you wouldn't have uninitialized variables dangling.

Reply:
You want to set these two variables and then immediately set them a second time every time this function is called for a fullscreen window?

Reply:
```c++
int targetWidth;
```
declares an uninitialized variable.
```c++
int targetWidth = base_width;
```
declares an initialized variable.

We aren't talking wasting CPU cycles or potential for optimization here; we are talking about removing the complexity of an uninitialized variable. You are going to do the assignment in the full screen case regardless.